cloudify_agent/shell/commands/daemons.py

```diff
@@ -199,10 +209,10 @@ def create(**params):
     click.echo('Creating...')
 
     _create_rest_ssl_cert(attributes)
+    _create_broker_ssl_cert(attributes)
```

Comment:
don't we want this to run after `get_broker_configuration`? `broker_ssl_cert` might be retrieved there